reconcile/utils/slack_api.py

```diff
@@ -98,6 +99,17 @@ def get_random_deleted_user(self):
                       'empty usergroup will not work')
         return ''
 
+    def get_user_id_by_name(self, user_name):
+        config = get_config()
+        mail_address = config['smtp']['mail_address']
+        result = self.sc.api_call(
+            "users.lookupByEmail",
+            email=f"{user_name}@{mail_address}"
+        )
+        if not result['ok']:
+            raise Exception(result['error'])
```

Comment:
Please, report at least the user name that is having the error, ideally also their email address. Also, I'd appreciate it if we raised either `ValueError` or a custom `SlackApiError`

Reply:
i've added a custom exception. about reporting the user name, this is raised when someone runs the command with a specific user, so i'd expect it is understood what is the problematic user?